django/db/backends/postgresql/base.py

```diff
@@ -86,6 +86,14 @@ def _get_varchar_column(data):
     return "varchar(%(max_length)s)" % data
 
 
+def _get_numeric_column(data):
+    if data["max_digits"] is None and data["decimal_places"] is None:
+        return "numeric"
+    elif data["decimal_places"] is None:
+        return "numeric(%(max_digits)s)" % data
```

Comment:
I believe I can delete this without a test failing, is there a missing test case here?